CONTRIBUTING.md

```diff
@@ -67,19 +71,13 @@ pre-commit install
 After the above, your `git commit` command will automatically trigger pre-commit
 checks, which are static code analysis tools we use.
 
-### Run statis analysis by hand (without using pre-commit)
```

Comment:
I suggest we keep this in case people want to run them by hand. We can note this is not preferred method.

Reply:
this can become cumbersome to maintain as we add continue more things to `pre-commit`.

Reply:
i'll add it back and mark it as deprecated.

Reply:
done